.github/workflows/pull-request.yml

```diff
@@ -4,33 +4,43 @@ on: [push, pull_request]
 
 jobs:
   ubuntu-build:
-
     runs-on: ubuntu-latest
-
     steps:
-      - uses: actions/checkout@v1
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
-        with:
-          java-version: 11
-      - name: Build with Gradle
-        env:
-          packageUser: ${{ github.actor }}
-          packagePAT: ${{ secrets.GITHUB_TOKEN }}
-        run: ./gradlew build
+      -   uses: actions/checkout@v1
+      -   name: Set up JDK 11
+          uses: actions/setup-java@v1
+          with:
+            java-version: 11
+      -   name: Build with Gradle
+          env:
+            packageUser: ${{ github.actor }}
+            packagePAT: ${{ secrets.GITHUB_TOKEN }}
+          run: ./gradlew build --scan --no-daemon
+      -   name: Archive Error Log
+          uses: actions/upload-artifact@v2
+          if: failure()
+          with:
+            name: Ballerina Internal Log
+            path: email-ballerina/ballerina-internal.log
+            if-no-files-found: ignore
+      -   name: Archive Code Coverage JSON
+          uses: actions/upload-artifact@v2
+          with:
+            name: Code Coverage JSON
+            path: email-ballerina/target/report/test_results.json
+            if-no-files-found: ignore
 
-  windows-build:
+    windows-build:
+      runs-on: windows-latest
 
-    runs-on: windows-latest
-
-    steps:
-      - uses: actions/checkout@v1
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
-        with:
-          java-version: 11
-      - name: Build with Gradle
-        env:
-          packageUser: ${{ github.actor }}
-          packagePAT: ${{ secrets.GITHUB_TOKEN }}
-        run: ./gradlew.bat build
+        steps:
```

Comment:
`runs-on` and `steps` should be in the same level.

Reply:
Fixed.